Site-Reliability-Engineering/L2-Site-Reliability-Engineer.md

```diff
@@ -0,0 +1,67 @@
+# L2: Site Reliability Engineer
+
+> _I am a valuable and enthusiastic member of an Octopus team. I can work independently on small projects, and am focused on learning with a trajectory of growth into the contribution of a Senior Site Reliability Engineer or beyond._
+
+- **Planning horizon**: Current and next cycle
+- **Impact radius**: Self plus some team mates
+- **Evaluation**: Manager
+- **Responsibility and direction needed**: Weekly supervision needed from their manager or more senior engineers.
+
+## 🦉 Domain expertise
+
+- I am competent in my domain.
+- I am able to work independently on small projects.
+- I identify and raise potential problems with my peers or lead before getting blocked.
+
+<details>
+<summary>Examples</summary>
+
+- I completed maintenance work in a complex area of our systems, relying on my own knowledge and 
```

Comment:
can we adjust please to "relying on my own knowledge of networking and systems"

Reply:
as above, I think specific mention of networking, infrastructure and systems is needed here. This gives more focus on the specifics of SRE, compared to SWE.

Reply:
Thanks, I've updated